bin/cloud_builder.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
```

Comment:
It feels to me that this file should go to https://github.com/istio/global-build

Reply:
I can see the argument that cloud_builder.sh spans multiple repos and thus might not be suitable for placing in istio repo, but overall the build mostly pertains to the istio repo and it seemed beneficial to keep the file with the majority of the rest of the build files so that they're kept together and there's fewer issues with keeping repos and branches consistent.  We're also not sure yet how many repos will survive in the long term.

Reply:
We don't know what is going to happen tomorrow, we only know what we have today. Everything that is closely related to, release, global build, repo, and cloud builder should go to a different repo.